library.properties

```diff
@@ -0,0 +1,10 @@
+name=Azure SDK for C
+version=1.0.0-beta.1
+author=Microsoft Corporation
+maintainer=Microsoft <iotcert@microsoft.com>
+sentence=Azure SDK for C library (1.3.0-beta.1) for Arduino.
```

Comment:
Verify and update the email alias to one that is relevant to our team. Not sure if iotcert works.

Reply:
Updated to 
maintainer=Microsoft Azure IoT <aziotarduino@microsoft.com>